Assets/_Scripts/InputHandler.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class InputHandler : MonoBehaviour
+{
+    public static event Action<Vector3> MovementDirection;
+    public static event Action<Vector3> CloneCreator;
+    public static event Action ClonePreview;
+
```

Comment:
I shall build my living room in this space

Reply:
I admire that youre getting preapring for the unaffordable housing in metropolitans